### DISCUSSION TITLE
Как внутри SQL-ного скрипта дать путь до файла запроса на XPath & XQuery?

### DISCUSSION THREAD

**Discussion by tsv19su254052** [tsv19su254052/LoadWorkData-GUIs-and-Utilities#110] [Show and tell]
Найти способ не вставлять исходник файла запроса на **XPath & XQuery** в строке ввода в диалоге хранимой процедуры внутри `Management Studio` через буфер обмена, а выбирать его в диалоге открытия файла или дать URL до него `file:///P:/...`.

_Originally posted by @tsv19su254052 in https://github.com/tsv19su254052/LoadWorkDataAirFlightsDBNew/issues/1#issuecomment-1165599767_

**Comment by jvxram** [comment]
в принципе вопрос простой, но XPath и XQuery думали применять для парсинга сайтов. На остальное (SAX-парсинга XML-ных полей и файлов внутри баз данных - чисто XML-ных и смешанных) как-то не хватило фантазии ...
